_test_/first.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../src/App';
+
+describe('first test', () => {
+  const wrapper = shallow(<App />);
+  it('should render the App', () => {
+    expect(wrapper.length).toEqual(1);
+    expect(wrapper.find('div')).toHaveLength;
+  });
+});
```

Comment:
Please put an extra line at the end of this file.

Reply:
Okay. I have done this.